packages/cli/test/integration/shared/test-modules.ts

```diff
@@ -0,0 +1,11 @@
+import { Container } from '@n8n/di';
+
+import { ModuleRegistry } from '@/modules/module-registry';
+
+export async function load(moduleNames: string[]) {
+	for (const moduleName of moduleNames) {
+		await import(`../../../src/modules/${moduleName}/${moduleName}.module`);
```

Comment:
This will be cleaner once integration helpers move to `@n8n/integration-test-utils`